charts/spire/charts/spire-server/values.yaml

```diff
@@ -389,6 +389,12 @@ tornjak:
     #   cpu: 100m
     #   memory: 128Mi
 
+plugins:
+  KeyManager: {}
+  NodeAttestor: {}
+  UpstreamAuthority: {}
+  Notifier: {}
```

Comment:
This doesn't align with the naming conventions.

https://helm.sh/docs/chart_best_practices/values/#naming-conventions

Reply:
Do we want simple and smaller/manageable code, or do we want pretty but with extra complexity? We already established that we can't have both. Just having these here at all mandated more complexity in the templating to remove the blank ones because spire can't tolerate blank ones configured.

Reply:
@marcofranssen can you please confirm if you'd like the added complexity to do the lowercase first character conversion?

Reply:
Is the problem here that the first letter is upper case instead of lower case? Or something else?

Reply:
yes conventions for values are camelCase. We have one issue here though. Those things like KeyManager and NodeAttestor should not be in the values file over here as they are part of the builtin plugins for which we already have an implementation.

Reply:
I'll change it to camelcase. Its going to bloat up the macro, which is not worth it IMO, but I care more about getting this thing finally implemented then some extra boiler plate code.

Reply:
I found a way to do it using some of the string manipulation functions that doesn't require a lot of mapping. Updated.